tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import { render, fireEvent, screen } from '@testing-library/react'
+import { vi } from 'vitest'
+import RadioButtonInputs from '~/components/radio-button-inputs/RadioButtonInputs'
+
+const mockedProps = {
+  items: [
+    { title: 'First Radio Title', value: 0 },
+    { title: 'Second Radio Title', value: 1 },
+    { title: 'Third Radio Title', value: 2 },
+    { title: 'Fourth Radio Title', value: 3 }
+  ],
+  onChange: vi.fn(),
+  title: 'Rating'
+}
+
+describe('RadioButtonInputs test', () => {
+  beforeEach(() => {
+    render(<RadioButtonInputs { ...mockedProps } />)
+  })
+
+  it('Should render radio buttons title correctly'),
+  () => {
```

Comment:
The syntax is wrong.
Must be: it('Should render radio buttons title correctly',  () => {

Reply:
fixed